l10n/en/products/vpn/more/ip-address.ftl

```diff
@@ -20,6 +20,7 @@ vpn-ip-address-does-it-travel = Does it travel with you?
 
 # Variables
 #   $url (url) - https://www.mozilla.org/products/vpn/more/when-to-use-a-vpn/
+vpn-ip-address-no-your-ip-v2 = No. Your IP address is only associated with one location unless you are using a VPN (we will get more into that later). When you are at your home and connecting to the internet you pay for, you are using one. However, if you check your email at home in the morning, then scan the news at a local coffee shop while waiting for your coffee, and then work from an office, you will have used different IP addresses at each location.
 vpn-ip-address-no-your-ip = No. Your IP address is only associated with one location unless you are <a href="{ $url }">using a VPN</a> (we will get more into that later). When you are at your home and connecting to the internet you pay for, you are using one. However, if you check your email at home in the morning, then scan the news at a local coffee shop while waiting for your coffee, and then work from an office, you will have used different IP addresses at each location.
```

Comment:
Nit: add a comment above this string to signal that it's outdated (these appear as metatata in Pontoon I believe).

```suggestion
# Outdated string
vpn-ip-address-no-your-ip = No. Your IP address is only associated with one location unless you are <a href="{ $url }">using a VPN</a> (we will get more into that later). When you are at your home and connecting to the internet you pay for, you are using one. However, if you check your email at home in the morning, then scan the news at a local coffee shop while waiting for your coffee, and then work from an office, you will have used different IP addresses at each location.
```

Reply:
thanks - good call!